docs/content/dagster-cloud/developing-testing/branch-deployments/using-branch-deployments-with-github.mdx

```diff
@@ -0,0 +1,320 @@
+2. In the **Security** section of the sidebar, click **Secrets > Actions**.
+3. Click **New repository secret**.
+4. In the **Name** field, enter the name of the secret. For example, `DAGSTER_CLOUD_API_TOKEN`
+5. In the **Value** field, paste the value of the secret.
+6. Click **Add secret**.
+
+Repeat steps 3-6 for each of the secrets required for your registry type:
+
+<details>
+  <summary><strong>AMAZON ECR</strong></summary>
+
+- `DAGSTER_CLOUD_API_TOKEN` - The Dagster Cloud agent token you created in [Step 1](#step-1-generate-a-dagster-cloud-agent-token)
+- `ORGANIZATION_ID` - Your Dagster Cloud organization ID
+- `AWS_ACCESS_KEY` - The **Access key ID** of the AWS IAM user you created in [Step 2](#step-2-create-and-configure-an-agent)
+- `AWS_SECRET_ACCESS_KEY` - The **Secret access key** of the AWS IAM user you created in [Step 2](#step-2-create-and-configure-an-agent)
+
+The **Actions secrets** page should look like the following:
+
+<Image
+alt="Configured GitHub Actions secrets for an AWS ECR registry"
+src="/images/dagster-cloud/developing-testing/branch-deployments/github-aws-ecr.png"
+width={1091}
+height={623}
+/>
+
+</details>
+
+<details>
+  <summary><strong>DOCKERHUB</strong></summary>
+
+- `DAGSTER_CLOUD_API_TOKEN` - The Dagster Cloud agent token you created in [Step 1](#step-1-generate-a-dagster-cloud-agent-token)
+- `ORGANIZATION_ID` - Your Dagster Cloud organization ID
+- `DOCKERHUB_USERNAME` - Your DockerHub username
+- `DOCKERHUB_TOKEN` - <!-- https://docs.docker.com/docker-hub/access-tokens/#create-an-access-token -->
+
+The **Actions secrets** page should look like the following:
+
+<Image
+alt="Required GitHub Actions secrets for a DockerHub registry"
+src="/images/dagster-cloud/developing-testing/branch-deployments/github-dockerhub.png"
+width={790}
+height={318}
+/>
+
+</details>
+
+<!--
+Google Cloud Registry
+  - GCR_JSON_KEY (https://cloud.google.com/container-registry/docs/advanced-authentication#json-key) -->
+
+### Step 4.3: Update GitHub Workflow files
+
+In this step, you'll update the GitHub Workflow files in the repository to set up Docker registry access.
+
+In the `.github/workflows/deploy.yml` and `.github/workflows/branch_deployments.yml` files, uncomment the `step` associated with your registry.
+
+For example, for an Amazon ECR registry, we'd uncomment the following portion of the workflow files:
+
+```yaml
+steps:
+  - name: Configure AWS credentials
+    uses: aws-actions/configure-aws-credentials@v1
+    with:
+      aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      aws-region: YOUR_REGISTRY_REGION
+```
+
+Save and commit the files to the repository.
+
+### Step 4.4: Verify Action runs
+
+The last step is to verify that the Action runs successfully.
+
+1. In the repository, click the **Actions** tab.
+2. In the list of workflows, locate the latest branch deployment run. For example:
```

Comment:
I think it would also be nice here to mention that clicking the run will populate the logs for the deployment. A screenshot here would be nice too.

I know I (and others) found it difficult to find these logs and it would be nice to explicitly point them out.

Reply:
Good call - I'll add that in.